v2/api/scheme.go

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package api
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/Azure/azure-service-operator/v2/internal/controllers"
+)
+
+// CreateScheme is a helper method to export 'internal/controllers/controller_resources.go/CreateScheme()' method outside internal package for external use.
```

Comment:
minor: This documentation should be public facing, so just say what the method does ("creates a runtime.Scheme containing all ASO resource definitions"). if you want to include a comment about the fact that it's exporting the internal function (and/or _why_ it does that), IMO put that bit of the comment inside the method so that it doesn't show up in the public facing documentation.

Reply:
Done!